lib/config/presets/npm.ts

```diff
@@ -1,11 +1,10 @@
 import { logger } from '../../logger';
-import { getDependency } from './get';
-import { Preset } from '../common';
+import { getDependency } from '../../datasource/npm/get';
 
 export async function getPreset(
   pkgName: string,
   presetName = 'default'
-): Promise<Preset> {
+): Promise<any> {
```

Comment:
Why this?

Reply:
`Preset` was just an alias for `any` anyway, so I couldn't be bothered creating a new `custom.ts` just for that.

Reply:
ok, but we should add a type for it later.

Reply:
I want to ge rid of almost any `any` in future

Reply:
Fixing it